css/style.css

```diff
@@ -0,0 +1,442 @@
+body {
+	font-family: 'GothamPro';
+	max-width: 1030px;
+	margin: 0 auto;
+}
+.header__menu {
+	height: 115px;
+	display: flex;
+	align-items: center;
+}
+.header__logo {
+    width: 55px;
+    height: 55px;
+    display: flex;
+    align-items: center;
+    flex: 0.1;
+    margin-right: 18px;
+}
+.menu {
+	flex: 1.9;
+	display: flex;
+	justify-content: space-around;
+}
+.menu__item {
+	color: #000;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5980773)